### DISCUSSION TITLE
MulticopterMotorModel explanation

### DISCUSSION THREAD

**Discussion by retif** [bitcraze/discussions#1110] [Q&A]
Hello, awesome project. Can you please explain how did you managed to calculate MulticopterMotorModel plugin parameters for sdf file? 

timeConstantUp, timeConstantDown, maxRotVelocity, motorConstant, momentConstant, rotorDragCoefficient, rollingMomentCoefficient, rotorVelocitySlowdownSim, motorType

Thank You!

**Comment by knmcguire** [comment]
For reference, this is the repo it is referring to: https://github.com/bitcraze/crazyflie-simulation for the gazebo files.


About the question, I believe the values originate from the thesis given in here: https://github.com/orgs/bitcraze/discussions/37. But it's been a while ago so I'll need to dig it up :)
